### ISSUE TITLE
Looking for a new maintainer

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/twofactor_gateway#454]
Hello fellow developers, admins and users,

I've not been able to give the app the attention it deserves due to some new responsibilities. I would therefore like to ask others who actively use the app to take over maintainership, like review PRs, evaulate bug reports, help stuck users and keep the app updated according to [the upgrade guide](https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/upgrade-guide.html).

Would anyone be interested in this? I'm happy to mentor you for a few months.

**Comment by ChristophWurst** [comment]
I'm still looking. If we don't find anyone then we'll have to set the repo to archive mode and possibly take the app off the app store. Leaving an unmaintained app out there that is supposed to improve people's security would be irresponsible. Hope that makes sense.

**Comment by sunjam** [comment]
Created a topic about this on the forum.  Thank you for developing such an excellent app!

https://help.nextcloud.com/t/two-factor-gateway-app-needs-a-new-maintainer-asap/124662

**Comment by rotdrop** [comment]
I don't think that I can find the time to do much. Keeping in sync with future Nextcloud versions would probably ok, but anything beyond bug-fixing and adding things I want to use by myself is more or less out of scope.

**Comment by ChristophWurst** [comment]
I'm unsubscribing from new tickets now. @vitormattos if you want to take over this app it's all yours :)

**Comment by vitormattos** [comment]
I can help with this app.

**Comment by vitormattos** [comment]
@StCyr I want but I was removed from Nextcloud organization at GitHub and because this I can't do any work at repositories of Nextcloud organization.

**Comment by vitormattos** [comment]
Hello @ChristophWurst,

As we can see, this app has not received maintenance for a while and currently has many open PRs. I’d like to volunteer to take care of it. I’ve already updated a fork to be compatible with the latest Nextcloud Server, but I would much prefer to keep the development in this official repository so that the community benefits directly.

Here is a test PR I created with many updates:

* https://github.com/LibreSign/twofactor_gateway/pull/1

I’m open to different options, for example:

* Becoming a maintainer here and submitting PRs to bring the app up to date, with the ability to publish new releases when needed.
* Or, if the team prefers, having the repository transferred so that I can continue maintenance under my account and coordinate new releases (including handling the API key for the app store).

My main goal is simply to ensure this app stays alive and available for Nextcloud users but I was removed from Nextcloud organization. I’m happy to follow any process you suggest to make this happen.

Thanks for considering this. I’m ready to help in whatever way is most useful.

**Comment by ChristophWurst** [comment]
Thank you @vitormattos! You should have an invite in your inbox ;-)

You're maintainer of this app now. I'm looking into the release pipeline right now and what I need to adjust.

**Comment by vitormattos** [comment]
Thanks a lot!

I will make more improvements before start the release flow. When it's possible, I'll contact you.

Then, I think that now we can finally close this issue.

**Comment by vitormattos** [comment]
Hello @ChristophWurst I think that now we have a stable version and is possible to go ahead with the process to launch a new release.

I and @rotdrop made some tests into the current version and I believe that it's a good moment to create the stable versions compatible with Nextcloud 31 and 32.

Did you have any suggestion about how we can make the release process of this app?
